src/deepsparse/loggers/build_logger.py

```diff
@@ -245,7 +246,19 @@ def build_data_loggers(
     data_loggers = []
     if not data_logging_config:
         return data_loggers
+
     for target_identifier, metric_functions in data_logging_config.items():
+        if target_identifier.endswith(PREDEFINED_IDENTIFIER_ALIAS):
+            """
+            If the target identifier ends with the predefined alias,
+            overwrite the current target identifier and metric functions with the one
+            retrieved from the predefined data logging confuguration
+            """
+            for target_identifier, metric_functions in metric_functions_from_predefined(
+                metric_functions, target_identifier[: -len(PREDEFINED_IDENTIFIER_ALIAS)]
+            ):
+                pass
```

Comment:
This may be a bit ugly but does the job.

Reply:
why is this necessary? if we only care about the last `metric_functions`, then why require the generator at all instead of just returning what we actually want?